ods_ci/tests/Resources/ServiceMesh.resource

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation       Main Service Mesh Resource file, to manage the operator and its resources
+
+Resource            OCP.resource
+Resource            ODS.robot
+
+Library             OpenShiftLibrary
+
+
+*** Variables ***
+${SERVICE_MESH_OPERATOR_NS}                 openshift-operators
+${SERVICE_MESH_OPERATOR_DEPLOYMENT_NAME}    istio-operator
+${SERVICE_MESH_CR_NS}                       istio-system
+
+${IS_PRESENT}                           0
+${IS_NOT_PRESENT}                       1
+
+
+*** Keywords ***
+Delete Service Mesh Control Plane
+    [Documentation]    Delete Service Mesh Control Plane
+    [Arguments]    ${namespace}        ${reconsile_wait_time}=15s
+    ${rc}   ${output}=    Run And Return Rc And Output
+    ...    oc delete ServiceMeshControlPlane data-science-smcp -n ${namespace}
+    Should Be Equal    "${rc}"    "0"   msg=${output}
+    # Allow operator time to reconsile
+    Sleep    ${reconsile_wait_time}
+
```

Comment:
Instead of a sleep, wait for the SMCP to be deleted to ensure that it actually is, [similar to this example](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/140__service_mesh.robot#L51).